packager/media/base/media_handler_test_base.h

```diff
@@ -145,8 +146,10 @@ MATCHER_P6(MatchEncryptionConfig,
            constant_iv,
            key_id,
            "") {
-  const std::string constant_iv_hex = absl::BytesToHexString(arg.constant_iv);
-  const std::string key_id_hex = absl::BytesToHexString(arg.key_id);
+  const std::string constant_iv_data(constant_iv.data(), constant_iv.size());
```

Comment:
#1143 has a new utility to make this work without copying the data first.  Let's land that PR, then rework this section to use that utility.

See https://github.com/shaka-project/shaka-packager/pull/1143/files#diff-14bc2abce24d1762d3fbf330540b87e2ae6b573103e47aeffbf2a87e89dde806

Reply:
Sure